javascripts/discourse/components/popular-tags.js

```diff
@@ -9,7 +9,25 @@ export default class PopularTags extends Component {
   constructor() {
     super(...arguments);
     const count = this.args?.params?.count || 10;
-    this.topTags = (this.site.get("top_tags") || []).slice(0, count);
+    let tags;
+    const excludedTags = this.args?.params?.excluded_tags || [];
+    const scope_to_category = this.args?.params?.scope_to_category || false;
```

Comment:
Can we use camelCase for this param? It matches better with our JS variable rules. 

So `scopeToCategory` instead of `scope_to_category`.